docs/courses/working_with_shape_plots.rst

```diff
@@ -0,0 +1,68 @@
+.. _working_with_shape_plots:
```

Comment:
add this file to a `:hidden:` `toctree` somewhere, presumably whichever file links to it... do the same for other added files